compiler/optimizations/removeUnnecessaryAutoCopyCalls.cpp

```diff
@@ -292,7 +292,7 @@ static void removePODinitDestroy()
     {
       // We expect both initCopy and autoCopy functions to have one argument
       // whose type is the same as the return type.
-      INT_ASSERT(fn->numFormals() == 1);
+      INT_ASSERT(fn->numFormals() >= 1);
```

Comment:
Actually it was some code that I've since reverted. I can put the assertion back, but this code doesn't actually care. It just needs to know that argument 1 is there. I think the comment is still reasonable and I'd like to leave the >= in the assert (because it's all this code actually cares about).